### PR TITLE
add dbc file for mazda_mx5_club_2017

### PR DESCRIPTION
**Pull request by yunglin**

### CODE REVIEW

**Comment by yunglin** [comment]
This dbc file shall applies to all current mazda model with skyactiv engine.

File: mazda_mx5_club_2017.dbc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+BO_ 1169 CONSTANTS_3: 8 XXX
+
+BO_ 358 NEW_MSG_6: 8 XXX
+ SG_ NEW_SIGNAL_2 : 7|8@0+ (1,0) [0|255] "" XXX
+ SG_ NEW_SIGNAL_1 : 23|16@0+ (1,0) [0|65535] "" XXX
+ SG_ NEW_SIGNAL_3 : 47|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 628 LOCK_STATUS: 8 XXX
+
+BO_ 832 SAFTY: 8 XXX
```
Comment (rbiasini) [inline mazda_mx5_club_2017.dbc]: misspelled?

File: mazda_mx5_club_2017.dbc [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+ SG_ CLUTCH_RELATED : 16|1@0+ (1,0) [0|1] "" XXX
+ SG_ GEAR_N : 36|1@0+ (1,0) [0|1] "" XXX
+ SG_ GEAR : 37|3@1+ (1,0) [0|255] "" XXX
+ SG_ GEAR_RELATED : 48|4@1+ (1,0) [0|1] "" XXX
+
+BO_ 514 POWERTRAIN_DATA: 8 XXX
+ SG_ Throttle : 39|16@0+ (0.002778,0) [0|1] "%" XXX
+ SG_ ENGINE_RPM : 7|16@0- (0.25,0) [0|8500] "" XXX
+ SG_ VehicleSpeed : 23|16@0+ (0.01,0) [0|1000] "km/h" XXX
+ SG_ NEW_SIGNAL_2 : 56|8@1+ (1,0) [0|255] "" XXX
```
Comment (rbiasini) [inline mazda_mx5_club_2017.dbc]: different endianess?

File: mazda_mx5_club_2017.dbc [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+ SG_ NI1_NI1 : 39|16@0+ (1,0) [1|100000] "" XXX
+ SG_ NI1_Counter : 7|8@0+ (1,0) [0|65535] "" XXX
+
+BO_ 576 STEERING_SENSOR: 8 XXX
+ SG_ STEERING_ANGLE : 7|16@0+ (0.01,-327.68) [-90|90] "deg" XXX
+ SG_ SteeringCorrSignalX : 39|16@0+ (0.01,-327.68) [0|50002] "" XXX
+
+BO_ 533 WHEEL_SPEED: 8 XXX
+ SG_ WHEEL_SPEED_FL : 7|16@0+ (1,-10000) [0|1] "kph" XXX
+ SG_ WHEEL_SPEED_FR : 23|16@0+ (1,-10000) [0|15000] "kph" XXX
```
Comment (rbiasini) [inline mazda_mx5_club_2017.dbc]: max value 15000?

**Comment by rbiasini** [comment]
@yunglin , Generally ok. it would be great if you can give it one more read, fixing capitalization consistencies, misspells etc... . Then I'll merge.

**Comment by pd0wm** [comment]
@yunglin, if you address the comments we can merge the PR. Especially the wheel speed speed factor and offset look weird.

**Comment by rbiasini** [comment]
no response, closing.

**Pull request closed without merging**